packages/plugins/plugin-adjust/src/AdjustPlugin.tsx

```diff
@@ -16,9 +16,11 @@ export class AdjustPlugin extends DestinationPlugin {
   type = PluginType.destination;
   key = 'Adjust';
 
+  private adjust: AdjustConfig | undefined = undefined;
```

Comment:
nit: `undefined` is the default, so no need to assign it

Reply:
Not sure I understand. Removing `undefined` gives me an error 

```Property 'adjust' has no initializer and is not definitely assigned in the constructor```